### PR TITLE
Onboard repository in Buildkite

### PR DESCRIPTION
**Pull request by mrodm**
## What does this PR do?

This PR onboards this repository into Buildkite and creates the skeleton for the main pipeline.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

-

### CODE REVIEW

**Comment by mrodm** [comment]
@jlind23 @alexsapran do we need to backport this PR to another version ?

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: beats
+
```
Comment (alexsapran) [inline catalog-info.yaml:8]: should we add the description
```suggestion
  description: :tropical_fish: Beats - Lightweight shippers for Elasticsearch & Logstash 
```

**Review by alexsapran — COMMENTED** [review]
we might want to set backport targets for the active branches here.

**Review by jlind23 — APPROVED** [review]

**Review by faec — APPROVED** [review]

**Comment by jlind23** [comment]
@mrodm just added backport labels for 8.8 and 7.17

**Pull request merged**